tools/functional-tester/etcd-tester/tester.go

```diff
@@ -239,6 +246,13 @@ func (tt *tester) startStressers() error {
 			return err
 		}
 	}
+
+	for _, ls := range tt.cluster.LeaseStressers {
```

Comment:
Why should these be treated any differently from the other stressers? Please put them in the `Stressers` list